returnn/config.py

```diff
@@ -122,7 +139,44 @@ def get(self, key, default=None):
             return self.post_config[key]
         return self.config.get(key, default)
 
+    def _write_network_stages(self, config_path):
+        """
+        write the networks of the staged network dict into a "networks" folder including
+        the access dictionary in the init file
+
+        :param str config_path:
+        """
+        config_dir = os.path.dirname(config_path)
+        network_dir = os.path.join(config_dir, "networks")
+        if not os.path.exists(network_dir):
+            os.mkdir(network_dir)
+
+        init_file = os.path.join(network_dir, "__init__.py")
+        init_import_code = ""
+        init_dict_code = "\n\nnetworks_dict = {\n"
+
+        for epoch in self.staged_network_dict.keys():
+            network_path = os.path.join(network_dir, "network_%i.py" % epoch)
+            pp = pprint.PrettyPrinter(indent=1, width=150, **self.pprint_kwargs)
```

Comment:
The handling of un-pretty-printable data is anyway not working for the network definition, so I guess it is okay to skip that. But then should I change the indent to 2 here or rather to 1 at the other location. In my opinion 2 makes no sense, as this only adds a single space after opening tags. This is not really about constant line indentations anyway...

Reply:
Yeah, but it would be good to have one common way of serializing the config and not re-implement it here. I don't really care about the indent here, more about the code duplication.

Reply:
I tried to fix this, but it does not make sense. The pretty printer can not be a member, as it is not serializable, and the rest differs. I tried to clean the rest a little bit, I hope that is sufficient.